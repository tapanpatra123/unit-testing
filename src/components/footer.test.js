import React from 'react';
import { shallow } from 'enzyme';
import Footer from './footer';
import { shallowToJson} from 'enzyme-to-json';

describe("footer",()=>{

    let wrapper;
    
    const props = {
        name : "footer contenet goes here"
    }

    beforeEach(()=>{
        wrapper = shallow(<Footer {...props}/>)
    });

    it("for snapshot",()=>{
        expect(shallowToJson(wrapper)).toMatchSnapshot()
    })

    it("should render footer title",()=>{
        const gotoText = wrapper.find(".footer").text();
        expect(gotoText).toBe("footer contenet goes here");
    })
})


// describe("footer",()=>{
    // it("for snapshot",()=>{
    //     const wrapper = shallow(<Footer name= {"Footerr"}/>);
    //     expect(shallowToJson(wrapper)).toMatchSnapshot()
    // })

    // it("should render footer title",()=>{
    //     const wrapper = shallow(<Footer name= {"Footerr"}/>);
    //     const gotoText = wrapper.find(".footer");
    //     expect(gotoText.text()).toBe("Footerr");
    // })
// })



