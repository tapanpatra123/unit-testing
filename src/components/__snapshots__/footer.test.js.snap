// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`footer for snapshot 1`] = `
<div
  className="footer"
>
  footer contenet goes here
</div>
`;
